compiler-cli/src/format.rs

```diff
@@ -79,7 +79,7 @@ pub fn unformatted_files(files: Vec<String>) -> Result<Vec<Unformatted>> {
         })?;
 
         if path.is_dir() {
-            for path in crate::fs::gleam_files_excluding_gitignore(&path) {
+            for path in crate::fs::gleam_files(&path) {
```

Comment:
Just to make you aware: the formatter will now format `.gitignored` files as well (although not the build directory). Let me know if this should be changed.